src/Microsoft.Management.Infrastructure.CimCmdlets/NewCimSessionOptionCommand.cs

```diff
@@ -29,7 +29,6 @@ public enum ProtocolType
     /// can subsequently use to create one or more CimSession connections. The
     /// options object holds the CIM Session information that is less commonly set
     /// and used by the IT Pro, and most commonly defaulted.
-    ///
```

Comment:
It seems that this line makes it more readable as there's two paragraphs

Reply:
There are many such places. Since I used automatic replacement, it is really impossible to fix it everywhere.

Reply:
Is it possible to craft a regex so that if the content of the last newline doesn't end with \> then keep the whitespace?

Reply:
Yes, I close the PR and open new one #7401 with the new regex.